native-oci-generative-ai/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.oracle.labs</groupId>
+  <artifactId>generative-ai</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>Oracle Cloud Infrastructure - Generative AI Demo</name>
+  <description>This project contains an example of using Oracle Cloud Infrastructure to generate text.</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>21</maven.compiler.source>
```

Comment:
Here Java version is 21, while in the workflow it's 17. They should match, because the action is failing: 
`Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project generative-ai: Fatal error compiling: error: invalid target release: 21 -> [Help 1] `